src/core/plugins/core-commands/server/index.ts

```diff
@@ -0,0 +1,27 @@
+import * as alt from 'alt-server';
+import { PluginSystem } from '../../../server/systems/plugins';
+
+// Add Command Imports here.
+import './commands/moderator/currency';
+import './commands/moderator/garage';
+import './commands/moderator/interior';
+import './commands/moderator/inventory';
+import './commands/moderator/noclip';
+import './commands/moderator/player';
+import './commands/moderator/teleport';
+import './commands/moderator/test';
+import './commands/moderator/vehicle';
+import './commands/moderator/wanted';
+import './commands/moderator/weapon';
+import './commands/moderator/chat';
+
+import './commands/player/chat';
+import './commands/player/death';
+import './commands/player/job';
+import './commands/player/seatbelt';
+import './commands/player/vehicle';
```

Comment:
Maybe create an `index.ts` file and then just do
```ts
import './commands/player';
```
same for moderator to keep imports `clean`